Chequeo de avance de dibujos técnicos

PiedraMorenoAlitza opened the pull request.

PiedraMorenoAlitza committed:
actualizacion

PiedraMorenoAlitza committed:
Chequeo de dibujos tecnicos

PiedraMorenoAlitza committed:
actualizacion

LAAngeles commented:
Los esquemas tu los hiciste? de lo contrario tienes que agregar la referencia, si tu los hiciste agrega tu nombre en el dibujo al igual que las unidades por ejemplo que las medidas están en milímetros o en pulgadas

PiedraMorenoAlitza commented:
Ok profe yo les coloco los detalles

PiedraMorenoAlitza committed:
Revision de dibujos

PiedraMorenoAlitza committed:
Revision de dibujos

PiedraMorenoAlitza committed:
Checar avance

PiedraMorenoAlitza committed:
Revision de Avance

PiedraMorenoAlitza committed:
Revision de avance

LAAngeles committed:
Update referencias.bib

al parecer realizaste cambios en otra carpeta que no era la tuya me refiero a la carpeta 19 y al archivo referencias.bib

LAAngeles committed:
Update referencias.bib

como realice cambios en el repositorio que esta en la nube tienes que escribir en la linea de comandos $ git pull para actualizar tu branch

LAAngeles commented:
Hola buenas tardes, por el momento voy a considerar correcto lo que te solicite pero posteriormente estaré siendo más puntual en como solicito la información. Buen trabajo felicitaciones.
